job-service-db/src/main/resources/db/migration/functions/public/R__createJobForDependencies.sql

```diff
@@ -81,6 +83,16 @@ BEGIN
         in_delay = 0;
     END IF;
 
+    -- Raise exception if any prerequisite jobs have failed
+    SELECT ARRAY_AGG(job_id) INTO failed_prerequisite_job_ids
+    FROM job
+    WHERE job_id = ANY(in_prerequisite_job_ids)
+      AND status = 'Failed';
+
+    IF array_length(failed_prerequisite_job_ids, 1) > 0 THEN
```

Comment:
Hi Dermot, I am following the same pattern as the existing validation in this file (`RAISE EXCEPTION...`), when I invoked the code though, I noticed that the SQL stack trace is also included after the error message  -is this a problem?

![image](https://github.com/JobService/job-service/assets/55391479/88e5b2d9-de68-4313-9772-3d404756397a)

Reply:
It's not ideal but as you rightly say this is the way all of the validations are done in this module so in my opinion you're right to do the same.